product-service/products/get-product-by-id.ts

```diff
@@ -1,19 +1,32 @@
 import * as AWSLambda from "aws-lambda";
 import { CORS_HEADERS } from "../cors-headers";
-import productList from "./products.mock.json";
+import { Client, ClientConfig, QueryResult } from "pg";
 
-interface Product {
-  id: string;
-  title: string;
-  description: string;
-  price: number;
-  count: number;
-}
+const { DB_HOST, DB_PORT, DB_DATABASE, DB_USERNAME, DB_PASSWORD } = process.env;
+const dbOptions: ClientConfig = {
+  host: DB_HOST,
+  port: +DB_PORT,
+  database: DB_DATABASE,
+  user: DB_USERNAME,
+  password: DB_PASSWORD,
+  ssl: { rejectUnauthorized: false },
+  connectionTimeoutMillis: 5000,
+};
```

Comment:
Do we need this config to be stored in all the handler files? Could we declare it once and then import it wherever we need it?

Reply:
good point! thank you!